x-pack/legacy/plugins/siem/public/lib/connectors/servicenow.tsx

```diff
@@ -153,7 +153,7 @@ const ServiceNowConnectorFields: React.FunctionComponent<ActionConnectorFieldsPr
               fullWidth
               isInvalid={isApiUrlInvalid}
               name="apiUrl"
-              value={apiUrl}
+              value={apiUrl || ''}
```

Comment:
is there a way to declare default values at the top of the component?
https://github.com/elastic/kibana/blob/d7118a19a87f8ed398b7d6c670f85092463529f9/x-pack/legacy/plugins/siem/public/lib/connectors/servicenow.tsx#L91 
maybe ```const { username = '', password ='' } = action.secrets;```?

Reply:
I update the source code with comments to explain why we do not have defaults.